Buddies.Web/.eslintrc.json

```diff
@@ -0,0 +1,20 @@
+{
+  "extends": [
+    "airbnb",
+    "airbnb-typescript",
+    "plugin:@next/next/recommended"
+  ],
+  "parserOptions": {
+    "project": "./tsconfig.json"
+  },
+  "rules": {
+    /* next.js addresses this */
+    "react/react-in-jsx-scope": "off",
```

Comment:
Generally, most people leave the react import in - it's a good habit to get into. But I'll leave it up to you.

Reply:
Next.js auto imports React, so this rule will not be necessary ([Next.js's ESLint plugin](https://github.com/vercel/next.js/blob/3dc4b524e79754804b8e53bd1783967d48ce5d13/packages/eslint-config-next/index.js#L18) also disables this, although we had to extend from a different config due to [documented incompatibilities](https://nextjs.org/docs/basic-features/eslint#recommended-plugin-ruleset)).